core/src/test/java/org/infinispan/remoting/TimeoutExceptionTest.java

```diff
@@ -0,0 +1,67 @@
+package org.infinispan.remoting;
+
+import org.infinispan.Cache;
+import org.infinispan.commands.write.PutKeyValueCommand;
+import org.infinispan.commons.CacheException;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.context.InvocationContext;
+import org.infinispan.interceptors.DDAsyncInterceptor;
+import org.infinispan.interceptors.locking.NonTransactionalLockingInterceptor;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.util.concurrent.TimeoutException;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.infinispan.test.TestingUtil.extractInterceptorChain;
+
+@Test(groups = "functional", testName = "remoting.TimeoutExceptionTest")
+public class TimeoutExceptionTest extends MultipleCacheManagersTest {
```

Comment:
I'm not convinced that a test is required to check the logs messages. 
But if you want to add a test, it will be faster to create the `Requests` and check the timeout exception.

Reply:
gonna remove it, i was already thinking that this is unnecessary/too much